charts/polymorphic-app/values.yaml

```diff
@@ -35,9 +35,9 @@ serviceTemplate:
 
   healthcheck:
     enabled: false
-    type: httpGet
-    path: /healthz
-    port: http
+    # type: httpGet
```

Comment:
Removing/commenting is not a solution? What's the problem? Can we not just use enabled: false and not render it?

Reply:
yes , for `health-check type` I have defined `livenessProbe and readinessProbe`  twice and differentiated with a if condition previously.  instead of moving hole probes to values.
due to the presence of those fields in the default  values `livenessProbe and readinessProbe`   are defining twice it was not giving any error when we deploy with helm but it giving an error with flux helmrelease
```
    # type: httpGet
```
```
    # type: tcpSocket
```